### PR TITLE
Bug Fixes and changes required for Advisor portal publishing

### PR DESCRIPTION
**Pull request by infosatheesh2020**

### CODE REVIEW

**Comment by infosatheesh2020** [comment]
@tsubasaxZZZ need your final review and approval before merge

**Comment by tsubasaxZZZ** [comment]
@infosatheesh2020 Thank you for your quick respond. I've checked through first pane to end pane quickly. It seems to be working correctly.

**Review by tsubasaxZZZ — APPROVED** [review]

**Pull request merged**
